enqueue/amqp-ext/0.3/etc/packages/enqueue_amqp.yaml

```diff
@@ -0,0 +1,23 @@
+enqueue:
+    transport:
+        default: 'amqp'
+        amqp:
+            host: '%env(ENQUEUE_AMQP_HOST)%'
+            port: '%env(ENQUEUE_AMQP_PORT)%'
+            login: '%env(ENQUEUE_AMQP_LOGIN)%'
+            password: '%env(ENQUEUE_AMQP_PASSWORD)%'
+            vhost: '%env(ENQUEUE_AMQP_VHOST)%'
```

Comment:
agreed here. PaaS expose credentials as a URL, so this config is not usable drop-in on such PaaS.